public/src/components/channelManagement/hooks/useValidation.ts

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from "react";
+import { ValidationStatus } from "../helpers/validation";
+
+const allValid = (validationStatus: ValidationStatus) => {
+  return Object.values(validationStatus).every((status) => status);
+};
+
+const useValidation = (onValidationChanged: (isValid: boolean) => void) => {
```

Comment:
Let's specify the return type for this function, so it's clear how the rest of the program uses this API:
```
type FieldValidationChange = (fieldName: string, isValid: boolean) => void;
const useValidation = (onValidationChanged: (isValid: boolean) => void): FieldValidationChange => { }...
```

Reply:
👍